django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -240,7 +240,7 @@ input[type="submit"], button {
     }
 
     .submit-row input.default {
-        margin: 0 0 5px 5px;
+        margin: 0 5px 5px 0;
     }
```

Comment:
This is now the same spacing as the non-responsive stylesheet, so this can be removed.